docs/application/native/guides/multimedia/openal.md

```diff
@@ -44,6 +44,11 @@ The main features of the OpenAL API include:
 
   > **Note**  
   > The device cannot fall into a sleep state while the context is not destroyed. To avoid unwanted battery consumption, destroy the context punctually.
+  >
+  > Since 3.0, extension alcDevicePauseSOFT() was introduced which allow applications to pause a playback device explictly.
```

Comment:
Please rephrase:
Since Tizen 3.0, introduction of extension `alcDevicePauseSOFT()`, which explicitly allows application to pause a playback.

Reply:
Hello, sabeena21. I have a question for this comment. It looks like this "sentence" doesn't have a verb. I feel awkward with it, so is it ok to go in this way?

Reply:
@mijin85cho, sure you can go with it.

Reply:
Minor change with allow->allows

Reply:
A minor issue. Please use ` around a word to highlight it.

\`alcDevicePauseSOFT()\` will become `alcDevicePauseSOFT()` on md.

Reply:
done